Tests/Build-Bicep.Tests.ps1

```diff
@@ -0,0 +1,49 @@
+try {
+    $ScriptDirectory = Split-Path -Path $PSCommandPath -Parent
+    Import-Module -FullyQualifiedName "$ScriptDirectory\..\Source\Bicep.psd1"
+}
+catch {
+    Throw "Unable to import Bicep module. $_"
+}
+
+InModuleScope Bicep {
+    BeforeAll {
+        $ScriptDirectory = Split-Path -Path $PSCommandPath -Parent
```

Comment:
Can't we just use $PSScriptRoot here?

Reply:
Maybe, I've just used the same as for all the other tests.